xml/System.Buffers.Text/Utf8Formatter.xml

```diff
@@ -60,14 +60,14 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="value" Type="System.Byte" />
-        <Parameter Name="buffer" Type="System.Span&lt;System.Byte&gt;" />
-        <Parameter Name="bytesWritten" Type="System.Int32&amp;" RefType="out" />
-        <Parameter Name="format" Type="System.Buffers.StandardFormat" />
+        <Parameter Name="value" Type="System.Byte" Index="0" />
```

Comment:
@joelmartinez @dend should mdoc be doing this? It seems that between Preview 1 and Preview 2 some signatures have changed but my expectation would be that parameter names would just be replaced instead of adding FrameworkAlternate values. Or should I perhaps first remove 2.1 assemblies and then add again on a different CI build?

Reply:
Hmm, strange ... so if I'm understanding, what's happened is that this parameter name changed between preview 1 and preview 2, so the original one (where the name was "buffer") no longer exists in the set of binaries that are being processed?

No, this is definitely an incorrect output ... I will make adjustments in mdoc and release a new build.

Reply:
@joelmartinez finally had some time to investigate and yes, I confirmed these were changes that happened between Preview 1 and 2 in API reviewer:
![image](https://user-images.githubusercontent.com/12971179/38853469-2de0d3ac-41d2-11e8-9c0d-97c8fb179e79.png)

Should I file a new issue? Should I try a workaround scuh as removing the assemblies and then adding the, back on a second CI run to unblock this work item?

Reply:
Ping @joelmartinez

Reply:
Ok @joelmartinez. I'll try that next. Filing the issue now.